README.md

```diff
@@ -1,2 +1,70 @@
-# service
+# Eik HTTP Service
+
 Eik REST API - As a standalone running HTTP service
+
+## Installation
+
+```
+npm install @eik/service
+```
+
+## Usage
+
+This server can either be run as a Node executable, or as a Fastify plugin.
+
+```sh
+npx @eik/service
+```
+
+For a production setup, the Fastify plugin method is recommended.
```

Comment:
How about splitting this into two?
one simple section showing how to start it, without a separate http server using `bin` and then another section where your mount the `.api()` as a Fastify plugin.